sdk/tables/perf-tests/data-tables/test/tables.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { PerfStressTest, getEnvVar, recordingPolicy } from "@azure/test-utils-perfstress";
+
+import { TableClient } from "@azure/data-tables";
+
+export abstract class TablesTest<TOptions = Record<string, unknown>> extends PerfStressTest<
+  TOptions
+> {
+  client: TableClient;
+  constructor(tableName: string) {
+    super();
+    const connectionString = getEnvVar("SAS_CONNECTION_STRING");
+    this.client = TableClient.fromConnectionString(connectionString, tableName);
+    this.client.pipeline.addPolicy(recordingPolicy(this.getRecordingClientV2()));
```

Comment:
Looks good! you can use the trick mentioned in https://github.com/Azure/azure-sdk-for-js/pull/16518#discussion_r680248482 to do the apply the same approach to other core v2-based packages.